packages/storybook/src/utils/createVisualRegressionStory.tsx

```diff
@@ -62,7 +62,10 @@ export const createVisualRegressionStory: CreateVisualRegressionStoryFn = (
  * @param props {@link React.PropsWithChildren}
  * @returns A {@link React.ReactElement}
  */
-export const VisualRegressionWrapper = ({ children }: PropsWithChildren): React.ReactElement => {
+export const VisualRegressionWrapper = ({
+  children,
+  className,
+}: PropsWithChildren<{ className: string }>): React.ReactElement => {
   const wrapperStyle: CSSProperties = {
```

Comment:
Dit hele zwikkie kan naar een aparte class in [`theme.css`](packages/storybook/src/styles/theme.css)

Reply:
done